spec/lib/fields/has_one_spec.rb

```diff
@@ -63,4 +63,45 @@
       expect(path).to eq("/fields/has_one/#{page}")
     end
   end
+
+  describe "#linkable?" do
+    context "when data is persisted" do
+      it "shows it" do
+        product_meta_tag = create(:product_meta_tag)
+        field = described_class.new(
+          :product_meta_tag,
+          product_meta_tag,
+          :show,
+        )
+
+        expect(field.linkable?).to be_truthy
```

Comment:
Fun thing we can do here:

```suggestion
        expect(field).to be_linkable
```